### DISCUSSION TITLE
Week 4 Assignment (Stata 1)

### DISCUSSION THREAD

**Discussion by hamzamajoka** [gui2de/ppol768-spring23#127] [Assignments Feedback]
@gui2de/ppol768-spring23 
Thank you for coming to the office hours today. I'll probably hold additional office hours on Monday morning for the Stata assignment. But you can also use this thread if you have any questions or comments regarding the assignment.

PS: Please do not use any user-written command for this assignment, especially striphtml  :)

**Comment by hamzamajoka** [comment]
@gui2de/ppol768-spring23 
Please reply here (with specific questions) if you want me to hold office hours later today. I might have a 1 hour slot @ 3 pm EST.

**Comment by emmanyhof** [comment]
I'm probably missing something, but is the dataset for Q4 supposed to be already scraped and in stata format? It's a .dta file but I'm not seeing any data when I import it

**Reply by hamzamajoka** [reply]
q4 dataset has only 1 variable and 1 row (i.e. all the scraped data is in that one cell.) You can use the following command to eyeball the data stored in that cell:

display s[1]

**Comment by hamzamajoka** [comment]
@gui2de/ppol768-spring23 

I'll be online in 10 mins in case anyone has any questions. You can use the following link: https://georgetown.zoom.us/my/gui2de

**Reply by hamzamajoka** [reply]
I've uploaded the recording of the office hours to the Box folder. (file name: 04c_office_hours_20230213) Best of luck!

**Comment by lgf94** [comment]
Hello, as mentioned yesterday I had finished the assignment, but now GitKraken is not letting me pull the branch with the updated assignment, it remains in local, hence I cannot do a pull request . I had pull the previous branch, done the request, been approved and I deleted the old branch. I will definitely go to office hours on tuesday to understand everything I have done wrong, but please lmk if you require revising the code beforehand and how to send it over to you. I did not send an email because you want us to use the discussion boards, but I cannot attach the do file since you do not want to publish some of the codes. thanks!
